src/components/timeline/ActiveTabTimeline.js

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is the significantly reduced version of the timeline. It doesn't have any global tracks yet. There is a TODO instead of its place.

Also I went with `timeline/ActiveTabTimeline.js` path but we can also change it to `timeline/active-tab/Timeline.js` or to `timeline/active-tab/index.js` but then they make the importing not easy to read and/or grep. Right now it's:
```
import ActiveTabTimeline from '../timeline/ActiveTabTimeline';
```
then it will be:
```
import ActiveTabTimeline from '../timeline/active-tab/Timeline.js';
```
which is not helpful for searching in the codebase. But maybe I'm overreacting it 😄

Reply:
I'm happy with your choice!